docs/src/quickstart.md

```diff
@@ -160,6 +161,63 @@ plot_energy(idata);
 gcf()
 ```
 
+### Additional information in Turing.jl
+With a few more steps, we can use Turing to compute additional useful groups to add to the [`InferenceData`].(@ref).
+
+To sample from the prior, one simply calls `sample` but with the `Prior` sampler:
+```@example quickstart
+prior = sample(param_mod, Prior(), nsamples; progress = false)
+```
+To draw from the prior and posterior predictive distributions we can instantiate a "predictive model", i.e. a Turing model but with the observations set to `missing`, and then calling `predict` on the predictive model and the previously drawn samples:
+```@example quickstart
+# Instantiate the predictive model
+param_mod_predict = turing_model(J, similar(y, Missing), σ)
+# and then sample!
+prior_predictive = predict(param_mod_predict, prior)
+posterior_predictive = predict(param_mod_predict, turing_chns)
+```
+And to extract the elementwise log-likelihoods, which is useful if you want to compute metrics such as [`loo`](@ref),
+```@example quickstart
+loglikelihoods = Turing.elementwise_loglikelihoods(param_mod, turing_chns)
+```
+This can then be included in the `from_mcmcchains` call from above:
+```@example quickstart
+# Convert into format compatible with ArviZ.jl
```

Comment:
It's a bit confusing as to whether this comment applies to the entire block or just the conversion of log-likelihoods. Could you clarify?

Reply:
Maybe we could just remove this line now, since we added the two comments below explaining what's going on?

Reply:
Yeah that works

Reply:
```suggestion
```